nltk/tokenize/punkt.py

```diff
@@ -1583,13 +1581,10 @@ def _second_pass_annotation(self, aug_tok1, aug_tok2):
             # starts a sentence or not.
             is_sent_starter = self._ortho_heuristic(aug_tok2)
 
-            if is_sent_starter == False:
+            if not is_sent_starter:
                 aug_tok1.sentbreak = False
                 aug_tok1.abbr = True
-                if tok_is_initial:
-                    return REASON_INITIAL_WITH_ORTHOGRAPHIC_HEURISTIC
-                else:
-                    return REASON_NUMBER_WITH_ORTHOGRAPHIC_HEURISTIC
+                return REASON_INITIAL_WITH_ORTHOGRAPHIC_HEURISTIC
```

Comment:
This seems off - why is the option for `REASON_NUMBER_WITH_ORTHOGRAPHIC_HEURISTIC` removed? is `tok_is_initial` always True? I wouldn't blame you for thinking that both branches of the if-else were identical, but there is a difference. Perhaps this is causing the CI errors.